src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
`print mut_slice` just shows the data pointer and the array's length. I looked at the pretty-printer's code as well, but I couldn't understand much :P